Write up on Ember CLI improvement tips (#591)
ember-learn/ember-blog: source/2020-04-03-the-ember-times-issue-142.md

---
title: The Ember Times - Issue No. 142
author: Abhilash, the crowd
tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
alias : "blog/2020/04/03-the-ember-times-issue-142.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

Ember CLI improvement tips 🚀,

READMORE

---

## [Ember CLI improvement tips 🚀](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2)

> [Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He showcases improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!```

Reviewer: ```suggestion
[Vasanth (@vasind)](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a GitHub gist. He showcased improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations in production. He also added tips on optimizing applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!
```

> [Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He talks about a bunch of improvements that include speeding up build times for development and test environments while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/), but no **spoilers** here, [head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!

Reviewer 2: ```suggestion
[Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He showcases improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!```
